Packet++/header/Layer.h

```diff
@@ -29,23 +29,26 @@ namespace pcpp
 		 */
 		virtual uint8_t* getDataPtr(size_t offset = 0) const = 0;
 
-		virtual ~IDataContainer() {}
+		virtual ~IDataContainer()
+		{}
 	};
 
 	class Packet;
 
 	/**
 	 * @class Layer
-	 * Layer is the base class for all protocol layers. Each protocol supported in PcapPlusPlus has a class that inherits Layer.
+	 * Layer is the base class for all protocol layers. Each protocol supported in PcapPlusPlus has a class that
+	 inherits Layer.
 	 * The protocol layer class expose all properties and methods relevant for viewing and editing protocol fields.
-	 * For example: a pointer to a structured header (e.g tcphdr, iphdr, etc.), protocol header size, payload size, compute
-	 * fields that can be automatically computed, print protocol data to string, etc.
-	 * Each protocol instance is obviously part of a protocol stack (which construct a packet). This protocol stack is represented
-	 * in PcapPlusPlus in a linked list, and each layer is an element in this list. That's why each layer has properties to the next and previous
-	 * layer in the protocol stack
-	 * The Layer class, as a base class, is abstract and the user can't create an instance of it (it has a private constructor)
-	 * Each layer holds a pointer to the relevant place in the packet. The layer sees all the data from this pointer forward until the
-	 * end of the packet. Here is an example packet showing this concept:
+	 * For example: a pointer to a structured header (e.g tcphdr, iphdr, etc.), protocol header size, payload size,
+	 compute fields that can be automatically computed, print protocol data to string, etc.
+	 * Each protocol instance is obviously part of a protocol stack (which construct a packet). This protocol stack is
+	 represented in PcapPlusPlus in a linked list, and each layer is an element in this list. That's why each layer has
+	 properties to the next and previous layer in the protocol stack.
+	 * The Layer class, as a base class, is abstract and the user can't create an instance of it (it has a private
+	 constructor).
+	 * Each layer holds a pointer to the relevant place in the packet. The layer sees all the data from this pointer
+	 forward until the end of the packet. Here is an example packet showing this concept:
```

Comment:
For some reason some of the lines don't begin with the `*` character...

Reply:
When the line is too long, clang-format split it. I manually fixed some, but this one was missed.

Reply:
fixed.